docs/docs/quickstart/installation/alternative-methods.md

```diff
@@ -2,4 +2,16 @@
 sidebar_position: 2
 ---
 
-# Alternative Installation Methods
\ No newline at end of file
+# Alternative Installation Methods
+
+## As a submodule
+
+Graaf can also be installed as a submodule in your project
```

Comment:
👍🏻

Reply:
Thank you so much 😊
This was the only alternate install method I could think of. Please suggest me if you know of any other methods. I'll be happy to add them as well.